Corona-Warn-App/src/main/AndroidManifest.xml

```diff
@@ -85,7 +85,6 @@
 
                 <data
                     android:host="e.coronawarn.app"
-                    android:pathPrefix="/"
```

Comment:
Why did this change?

Reply:
the url changed and does not have `/` a prefix fix leaving it prevents the app from responding to deeplinks